packages/camera/lib/camera.dart

```diff
@@ -227,10 +224,7 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
     try {
       _creatingCompleter = Completer<void>();
-      // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
-      // https://github.com/flutter/flutter/issues/26431
-      // ignore: strong_mode_implicit_dynamic_method
-      final Map<dynamic, dynamic> reply = await _channel.invokeMethod(
+      final Map<dynamic, dynamic> reply = await _channel.invokeMethod<Map<dynamic, dynamic>>(
```

Comment:
https://github.com/flutter/flutter/blob/5169ab5974571918121fa8a7ddb0d3e0623bc987/packages/flutter/lib/src/services/platform_channel.dart#L329

Reply:
cool, going to do a massive change on all the invokeMethod that has map type